### PR TITLE
Add Portland 2024 videos

### PR DESCRIPTION
**Pull request by plaindocs**
Super messy diff, but it *should* just be adding the youtube IDs.
I should probably look at not mangling the whitespace, but I can't really be bothered. 

----
📚 Documentation preview 📚: https://writethedocs-www--2153.org.readthedocs.build/

### CODE REVIEW

**Review by mxsasha — APPROVED** [review]
Looks like it didn't break the schedule or speaker page, so 👍🏻

**Comment by plaindocs** [comment]
Do we still have to futz to generate the content in `videos/*`? We do that manually right?

**Comment by mxsasha** [comment]
> Do we still have to futz to generate the content in `videos/*`? We do that manually right?

I think that is generated during a build with ENABLE_VIDEOS on.

**Comment by plaindocs** [comment]
@mxsasha take one more look please?

File: docs/conf.py [hunk 1]
```diff
@@ -183,14 +185,12 @@
...
-    if os.environ.get('MEETUP_API_KEY'):
-        try:
-            from _ext.meetup_events import main as meetup_main
-            html_context.update(meetup_main())
-        except:
-            print('Could not get meetup events.')
     html_context.update(videos.main())
 
+    # Fix legacy stuff
+    subprocess.run(['git', 'checkout', '--', 'videos/portland/2019/documentation-for-good-riona-macnamara.rst'])
```
Comment (mxsasha) [inline docs/conf.py]: 😅

**Comment by mxsasha** [comment]
LGTM :shipit:

**Comment by plaindocs** [comment]
Oh boo. That'll teach me.

@mxsasha 

Maybe I should just add the exception(s) in here instead https://github.com/writethedocs/www/blob/main/docs/_ext/videos.py#L136

**Comment by mxsasha** [comment]
> Maybe I should just add the exception(s) in here instead https://github.com/writethedocs/www/blob/main/docs/_ext/videos.py#L136

Feels like the best place, yes. Or just don't regenerate video files < 2024 at all there. Saves time too.

**Comment by plaindocs** [comment]
ok already

* At docs/_build/html/blog/newsletter-june-2019/index.html:139:

  internally linking to /videos/prague/2018/tackling-technical-debt-in-the-docs-louise-fahey, which does not exist

* At docs/_build/html/topics/index.html:421:

  internally linking to /videos/prague/2018/tackling-technical-debt-in-the-docs-louise-fahey/, which does not exist

* At docs/_build/html/videos/prague/2018/index.html:105:

  internally linking to /videos/prague/2018/tackling-technical-debt-in-the-docs-louise-fahey/, which does not exist

* At docs/_build/html/videos/prague/2018/index.html:108:

  internally linking to /videos/prague/2018/tackling-technical-debt-in-the-docs-louise-fahey/, which does not exist

**Comment by plaindocs** [comment]
@mxsasha any thoughts on an easy way to exclude certain talks here?

**Comment by mxsasha** [comment]
I don't understand why anything from Prague 2018 is being modified? Did we not publish that talk? But the video template file isn't removed in this PR? How does the build currently succeed on main?

If your question is just to stop that talk being included in the Prague 2018 videos index, i.e. removing the broken link, I think you can remove the youtubeId from 2018.prague.speakers.yaml. But I don't understand how anything in this PR is now breaking that link, if it is not broken on main.

**Comment by plaindocs** [comment]
Exactly. I think it's something about flakey linters getting upgraded and changing behavior. I'll try removing the Id and seeing if that helps.

**Pull request merged**